android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java

```diff
@@ -80,7 +84,12 @@
 import org.chromium.ui.base.DeviceFormFactor;
 import org.chromium.url.mojom.Url;
 
+import java.util.Arrays;
```

Comment:
I can't find where 
```
import java.util.Arrays;
import java.util.HashMap;
import java.util.Map;
```
looks like it is a leftover?

Reply:
yeap... removing